### ISSUE TITLE
Render audio with AkAudioEvents (Sequencer) (Unreal + PA)

### ISSUE THREAD

**Issue by AvivSham** [microsoft/ProjectAcoustics#62]
Hi All,
I’m trying to render a scene using a level sequencer and project acoustics (WWISE). Once the rendering process is finished it generates two files avi (video) and wave (audio) files but when I play the wav file I can hear only the sounds cues but not the audio acoustics (AkAudioEvents). How can I make UE render the audio acoustics in addition to the sound cues?
Adding AkAudioEvent to the sequencer didn’t help either.

OS:
UE 4.26
wwise 2021.1.1.7601
Windows 7 pro

@dvirsamuel

Thank you!

**Comment by MikeChemi** [comment]
Hi @AvivSham ,

I'm a little confused by the setup. A few questions for you...

You mention you're using both sound cues and AkAudioEvents -- does that mean you're using Unreal's audio engine alongside Wwise?

Project Acoustics requires you to use Acoustics Audio Components on the same Actor as the AkAudioEvents. Can you confirm you have Acoustics Audio Components on your actors?

How are you rendering the whole scene? Does your rendering process support Wwise without Project Acoustics?

The Project Acoustics Wwise mixer plugin is incompatible with Wwise 2021 at this time. It is possible to use Project Acoustics in Wwise without that mixer plugin, but I wanted to call that out specifically in case that's the source of the problem.

**Comment by dvirsamuel** [comment]
Hi @MikeChemi, thank you for your reply.
I am working with @AvivSham on the project.

TLDR: How can we render a simple scene (with ue and project acoustics) so we could hear the Acoustic Component actors in the final audio file (generated after rendering).

Our end goal is to render videos and record the audio in a ue4 environment. One common way to do that is with the Sequencer (which render scenes into movies and export a wav file with the audio).
We would like to record the audio that was computed and generated using project acoustics for the best sound experience and realistic audio.

We started by following the tutorial ([here](https://docs.microsoft.com/en-us/gaming/acoustics/unreal-integration)) and playing with the Sample Project (with SunTemplat level map).
We then created a simple scene with the Sequencer and added Acoustic Component with their AkAudioEvents into the Sequencer (see image below).

The problem is that no audio is being recorded and the generated wav file we get after rendering is silent.

For sanity check, we replaced the Acoustic Component with a simple Sound cue (which does not belong to wwise or project acoustics) and could hear the sound in the find audio file.


![image](https://user-images.githubusercontent.com/22732678/121208096-7b028b00-c882-11eb-8c8f-7615bc4abeea.png)


What could be the problem?

**Comment by MikeChemi** [comment]
Thanks for the additional information. We've never tried to use UE's Sequencer before, so you probably know more here than we do :) 

It sounds like a problem with hooking up Wwise to UE's Sequencer. I can't tell from your screenshot -- have you gone through AK's documentation about how to do this? In particular, I'm wondering if the audio events are being triggered and have a long enough duration.
https://www.audiokinetic.com/library/edge/?source=UE4&id=using_features_sequencer.html

If you're confident that your Sequencer setup is correct, try replacing the Acoustics Audio Components with AK Components (in essence, disabling Project Acoustics but still using Wwise) and see if you get audio that way.

**Comment by dvirsamuel** [comment]
Thank you very much! We will try 😄
